studio/src/rte/rte-link-editor.js

```diff
@@ -118,6 +126,34 @@ export class RteLinkEditor extends LitElement {
         return this.shadowRoot.querySelector('#linkHref');
     }
 
+    get #checkoutLink() {
```

Comment:
since this is a boolean, can we rename as `#isCheckoutLink `

Reply:
will do, thx

Reply:
any update?

Reply:
sorry, renamed now :)